src/PdoInterface.php

```diff
@@ -186,5 +186,5 @@ public function setAttribute(int $attribute, mixed $value): bool;
      * @return array
      *
      */
-    public static function getAvailableDrivers(): array;
+    public static function getAvailableDrivers();
```

Comment:
I missed this. Any reason why return type is removed ?

I am looking at https://www.php.net/manual/en/pdo.getavailabledrivers.php .

`public static PDO::getAvailableDrivers(): array`

Reply:
running unittests on php8.0 reveals this:
```
Fatal error: Declaration of PDO::getAvailableDrivers() must be compatible with Aura\Sql\PdoInterface::getAvailableDrivers(): array in Aura.Sql/src/AbstractExtendedPdo.php on line 0
```

the typehint is fine when running on php8.1

i guess i changed it to be compatible with `php8.0`

// reverting it

Reply:
You can stay 8.0 compatible by adding the following to AbstractExtendedPdo.php since its fine to be more strict in child classes.
`
    // Overwrite to support PHP 8.0
    public static function getAvailableDrivers(): array
    {
        return PDO::getAvailableDrivers();
    }
`